services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/api_client/_thin.py

```diff
@@ -57,66 +56,62 @@ def __init__(self, app: FastAPI):
 
     def _get_url(
         self,
-        dynamic_sidecar_endpoint: AnyHttpUrl,
+        dynamic_sidecar_endpoint: str,
```

Comment:
The same as in the above file. Please revert it everywhere in this file.

Reply:
done